webapp/game/avalon.py

```diff
@@ -108,6 +111,7 @@ def __init__(self, users):
         self.pk = self.avalon_game.pk
         self.quests = self.avalon_game.quests
         self.quest_master = self.avalon_game.quest_master
+        self.game_users = self.avalon_game.game_users
```

Comment:
L97, L114 needs to be cleaned up, maybe do a methodmissing type call to getattr and default to grabbing off self.game which is what this is doing anyways